main.ledger

```diff
@@ -2029,6 +2063,13 @@
     Liabilities:Reimbursement:Zach Latta
     ; Receipt: 552f1de496ded0f2e632ce6595239279.pdf
 
+2016/05/24 Google
+    Expenses:Services:Software                          $50.00
+    Liabilities:Reimbursement:Max Wofford
+    ; Receipt: efc7349a4868ab6a73b5605e644f135b.pdf
+    ; TODO doublecheck this is right category. this is google adwords
+    ; also is this ... negative money??
```

Comment:
No, cuz the other charge for 80something was not in parentheses. Very strange/inconsistent/strange